### PR TITLE
Register avro-schema-parser

### LINKED ISSUE #20: Unable to use Avro Schema Parser
When using `schemaFormat: "application/vnd.apache.avro;version=1.9.0"`, it fails with `PARSERS[String(...)] is not a function` error. It should be related to this issue https://github.com/asyncapi/parser-js/issues/157

To be able to validate AsyncAPI with Avro schema references, the https://github.com/asyncapi/avro-schema-parser has to be registered:

```
const avroSchemaParser = require('@asyncapi/avro-schema-parser')
parser.registerSchemaParser(avroSchemaParser);
```

### PR DESCRIPTION
**Pull request by VaclavChalupa**
fixes https://github.com/WaleedAshraf/asyncapi-github-action/issues/20

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+root = true
```
Comment (WaleedAshraf) [inline .editorconfig]: Add this to gitignore.

File: test/schema/light.command.MeasureLight.avsc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (WaleedAshraf) [inline test/schema/light.command.MeasureLight.avsc:1]: 👍

**Review by WaleedAshraf — COMMENTED** [review]
Looks good. Minor fix, remove the .editor file.

File: .editorconfig (hunk 1 above)
Comment (VaclavChalupa) [reply .editorconfig]: @WaleedAshraf actually, I committed it on purpose, IMO should be part of project with multiple participants: https://editorconfig.org/

Please confirm, if you want to remove it :)

**Comment by VaclavChalupa** [comment]
@WaleedAshraf hm, the check is failing here, any idea? Local `npm test` is working. Is there anything else (file...) what should be changed? Thx

**Comment by WaleedAshraf** [comment]
You can ignore the check. I think it's using the main branch to run the action. @HEm3R 
Once we merge this PR, I'll release a new version. That should fix it.

File: .editorconfig (hunk 1 above)
Comment (VaclavChalupa) [reply .editorconfig]: @WaleedAshraf removed, it could be create anytime, if needed 👍

**Comment by VaclavChalupa** [comment]
All notes reflected, ready to be merged 🤞

**Comment by WaleedAshraf** [comment]
Released https://github.com/WaleedAshraf/asyncapi-github-action/releases/tag/v0.0.8
The issue with the check was that we forgot to do the `npm run package`. It updates the dist/index file. All is good now. Thanks :)

**Pull request merged**